scripts/cray_runall.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+alps_present=`which alps`
+if [ $? == 0 ]; then
+launcher="aprun -n 1"
+else
+launcher="srun -n 1"
+fi
+echo Running fi_info
+$launcher ./fi_info 2>&1 | tee test.output
+if [ $? != 0 ] ; then
+exit -1
+fi
+count=`grep gni test.output | wc -l`
+if [ $count != 3 ] ; then
+echo "GNI doesn't appear in fi_info about, aborting..." 
+exit -1
+fi
+rm ./test.output
+sleep 1
+echo Running fi_ep_test
+$launcher ./fi_ep_test -f gni
+if [ $? != 0 ] ; then
+exit -1
```

Comment:
Okay, you'd like to have it run all of them?  I was thinking in terms of jenkins testing that it would be easier just to check for a non-zero return status from running the entire script, but I can modify it to give a list of pass/fails instead.